circle.yml

```diff
@@ -7,40 +7,41 @@ machine:
   hosts:
     testpilot.dev: 127.0.0.1
   node:
-    version: 6.2.0
+    version: 6.9.1
   environment:
     TESTPILOT_ENABLE_PONTOON_BRANCHES: master
 
 dependencies:
 
   cache_directories:
     - "addon/node_modules"
-    - "node_modules"
-
-  pre:
-    # https://discuss.circleci.com/t/nvm-commands-are-not-evaluated/943
-    - nvm use 6.2.0 && nvm alias default 6.2.0
+    # node_modules is included by default
 
   override:
-    - node --version
-    - npm --version
+    - npm install
```

Comment:
Might want to move these two commands into a `./bin/circleci/setup-node-dependencies.sh' script to be clear